.gitignore

```diff
@@ -26,3 +26,5 @@ temp/
 .shelf
 # Intellij default build output directory
 out/
+
+currentpid
```

Comment:
A new line is needed at the end.

Did you follow the contributor guide and set up Save actions plugin?